functional-tests/src/test/java/com/redhat/che/functional/tests/Che7DialogAbout.java

```diff
@@ -108,25 +108,25 @@ public void openDialogAbout() {
   }
 
   private TestWorkspace createChe7Workspace() {
-    prepareWorkspace("che7", WORKSPACE_NAME, null);
```

Comment:
why workspace name is a constant and stack / devfile name is harcoded ?

Reply:
The workspace name is being generated here: https://github.com/redhat-developer/rh-che/pull/1470/files#diff-84d9e98867a01b8453d3d350179e872dL39
And we are always using the same devfile

Reply:
@ScrewTSW please make devfile / stack a constant also.

Reply:
done